3-platform/order/spec/src/main/java/com/_6core/platform/orderspec/rest/v1/dto/OrderResponse.java

```diff
@@ -0,0 +1,12 @@
+package com._6core.platform.orderspec.rest.v1.dto;
+
+import java.math.BigInteger;
+import java.util.Set;
+
+public record OrderResponse(
```

Comment:
I didn't get the difference between `/orders` and `/orders/{orderId}/items` endpoints.
They both return `OrderResponse`. Pls, work on naming classes. 

/orders/ - is expected to return Order History (without details about which items were bought)
/orders/{orderId}/items - is expected to return Order Details (which items were bought).